src/DefaultBackground.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { withStyles, withStylesPropTypes } from 'react-with-styles';
+import {
+  BACKGROUND_HEIGHT_UNITS,
```

Comment:
I wonder if this constant can be instantiated in the theme itself (or maybe it should be added as a prop just to be configurable?)